src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -1239,7 +1239,8 @@ def test_osd_activate(self, _run_cephadm, cephadm_module: CephadmOrchestrator):
                     'tags': {
                         'ceph.cluster_fsid': cephadm_module._cluster_fsid,
                         'ceph.osd_fsid': 'uuid'
-                    }
+                    },
+                    'type': 'data'
                 }]
```

Comment:
Great, you needed to modify the tests already. Can you also add a test that contains `'type': 'db'`?